templates/epic.xml.jinja2

```diff
@@ -92,136 +88,79 @@
      - B0 tracker
      - Off-momentum tracker
      - Far forward roman pots
-     - Zero Degree Calorimeter
+     - Zero Degree Calorimetergit restore --staged
```

Comment:
This needs some attention

Reply:
The disadvantage of Lenovo track points...

Reply:
```suggestion
    - Zero Degree Calorimeter
```